### ISSUE TITLE
Support for visual studio 2017 "non preview"

### ISSUE THREAD

**Issue by mrpink76** [dgriffen/rls-vs2017#11]
Just updated Visual Studio 2017 to version 15.8 and there seem to be native RLS support.

Is the extension updated to work with this version of Visual Studio?

**Comment by dgriffen** [comment]
The extension should work just fine in 15.8+. I'll have to update the documentation to reflect that. Feel free to reopen this issue if I'm mistaken and it stopped working in 15.8 RTW.

**Comment by jeffvandyke** [comment]
I've just tested Visual Studio + Rust for the first time, and the extension is working great using stable Rust toolchain and Visual Studio Community 2017 Version 15.9.7 (latest stable as of Feb 2019). Release notes for Visual Studio 15.8 [here](https://docs.microsoft.com/en-us/visualstudio/releasenotes/vs2017-relnotes-v15.8) indicate that this was indeed when native LSP support was added.

I was wondering: when willl the README.md file (and the extension marketplace page) would be updated to reflect that preview versions of VS are no longer required - only 15.8 or newer?

(Another pleasant surprise was how polished Rust's (or LLVM's) .pdb support was for debugging - visualizing vectors, modifying locals, and setting breakpoints worked flawlessly when attaching to a process - perhaps we can steal some of [Visual Rust](https://github.com/PistonDevelopers/VisualRust)'s work to give a more streamlined experience out of the box? Separate issue perhaps.)
